config/common/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml

```diff
@@ -41,7 +41,9 @@ enable_nodelocaldns: false
 
 kubernetes_audit: true
 audit_log_path: "/var/log/audit/kube-apiserver-audit.log"
-audit_log_maxage: 7
+audit_log_maxage: 30
+audit_log_maxbackups: 10
+audit_log_maxsize: 100
```

Comment:
Do we need to comment on this change.
I mean do we know how much in terms of GB will be generated over 30 days?
I think that we on cleura cloud only have 30-40gb on the boot disk.